bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/link/LinkManagerImpl.java

```diff
@@ -121,4 +124,14 @@ private void initModel() {
         return new LinkBuilderImpl(url, request, pathProcessors, shadowingDisabled);
     }
 
+    /**
+     * Checks whether the provided URL string represents an external link.
+     *
+     * @param url an URL string
+     *
+     * @return {@code true} if the {@code url} is an external link, {@code false} otherwise.
+     */
+    public static boolean isExternalLink(String url) {
```

Comment:
I don't think it would be necessary to extend the link interface with that as long was we don't require that information in the public API. 

As long as we keep it an internal implementation detail the method can be anywhere and if needed be refactored in the future.

Reply:
As this method is currently only use by the ListImpl we could also just add a private method in the model itself. If we later see a common use case we can then have a public helper method for this or extend the Link interface.

Reply:
I added it to ListImpl initially and then I moved it to LinkManagerImpl for more visibility in case we would need this later outside of ListImpl. I can move it wherever we think it's more convenient.